src/handlers/legacy_redirect_handler.cr

```diff
@@ -16,8 +16,8 @@ class LegacyRedirectHandler
     "/guides/logging-and-error-handling"          => Guides::GettingStarted::Logging,
     "/guides/writing-json-apis"                   => Guides::JsonAndApis::RenderingJson,
     "/guides/json-and-apis/handling-errors"       => Guides::HttpAndRouting::ErrorHandling,
-    "/guides/generating-test-data"                => Guides::Database::Testing,
-    "/guides/browser-tests"                       => Guides::Frontend::Testing,
+    "/guides/generating-test-data"                => Guides::Testing::CreatingTestData,
```

Comment:
I think we need to al add separate entries for the now dead links for `https://luckyframework.org/guides/frontend/testing` and `https://luckyframework.org/guides/database/testing`